system-tests/e2e-test-fixtures/build.gradle.kts

```diff
@@ -0,0 +1,33 @@
+/*
+ *  Copyright (c) 2022 Microsoft Corporation
```

Comment:
is this correct?

Reply:
Yes I think this is. This file was initially in the `testFixtures/tests` folder.